### ISSUE TITLE
Basic pages

### ISSUE THREAD

**Issue by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#286]
Basic pages should be able to use layout builder OR have a useful template.

**Comment by michael-wynne-wsu** [comment]
@christa-kavanagh-wsu @joelle-birano-wsu I'm working on this. What kind of section layouts and blocks would you like to have available when making a page?

**Comment by christa-kavanagh-wsu** [comment]
Is there a simple way to add graphic elements? Like custom section dividers, etc.?

**Comment by michael-wynne-wsu** [comment]
Depends what exactly you have in mind. Everything is custom at this point.

**Comment by joelle-birano-wsu** [comment]
I think Iʻd add 
- Local contexts labels with descriptions
- featuerd content

**Comment by michael-wynne-wsu** [comment]
For LC, a picker like in content, or something else?

**Comment by christa-kavanagh-wsu** [comment]
> Depends what exactly you have in mind. Everything is custom at this point.

I would just see if it's possible to add a section divider that has the option for users to upload their own graphic, similar to how they can upload their own logo and favicon.

**Comment by michael-wynne-wsu** [comment]
> I would just see if it's possible to add a section divider that has the option for users to upload their own graphic, similar to how they can upload their own logo and favicon.

To me that sounds like a broader theming setting, not just a layout builder block.

**Comment by joelle-birano-wsu** [comment]
Yeah I think a label and project picker like we have in content, but that will display the labels in a grid. I imagine it as a space for people to introduce Local Contexts and talk a bit about how theyʻre using labels and notices, and then show the labels and either include the actual descriptions, or add shorter summaries below.

<img width="668" height="621" alt="Image" src="https://github.com/user-attachments/assets/35f6c36c-2220-46b8-8baf-bbbf5ca7150c" />

**Comment by michael-wynne-wsu** [comment]
Re: LC that was kind of the point of the site and group registry pages. Not that we can't, I just wonder if it's too duplicative.

**Comment by joelle-birano-wsu** [comment]
I wondered that too...Iʻm just thinking about visitors who may have no concept of Local Contexts or Indigenous data sovereignty - It might be helpful for them to have something on the about page that says "Hey here are these labels and this is why theyʻre there" - just to give them broader context. I guess folks could just build that on their own with the layout features we are already including...

**Comment by michael-wynne-wsu** [comment]
There are existing LB related modules too, but I'm not sure how broadly used many of them are... https://www.drupal.org/project/layout_builder/ecosystem
